src/lib/Microsoft.Health.Logger/Telemetry/ITelemetryLogger.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.Health.Logging.Telemetry
 {
     public interface ITelemetryLogger
     {
-        void LogMetric(Metric metric, double metricValue);
+        void LogMetric(Metric metric, object metricValue);
```

Comment:
>object [](http://example.com/codeflow?start=38&length=6)

What are some examples of none double values?

@wi-y is this supported via MDM?

Reply:
As mentioned in the other comment, I thought to update this since the underlying TelemetryClient api supported passing in object and figured we could expose passing in other types too if needed. But happy to change it back if we want to restrict based on MDM limitations.